openapi/src/parts/workspace.yaml

```diff
@@ -416,12 +425,15 @@ components:
 
     WorkspaceDescription:
       type: object
-      required: [id]
+      required: [id, userFacingId]
       properties:
         id:
           description: The ID of the workspace
```

Comment:
document that this one is immutable maybe.

Reply:
Done